### PR TITLE
bash: Update some options

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: Zhou Hao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Thanks,
LGTM

File: contrib/completions/bash/buildah [hunk 1]
```diff
@@ -164,6 +164,8 @@ return 1
          --version -v
          "
      local options_with_args="
+         --registries-conf
```
Comment (TomSweeneyRedHat) [inline contrib/completions/bash/buildah]: Do we need these here since they're in the global options below?

**Comment by TomSweeneyRedHat** [comment]
@q384566678 thanks for the updates.  I think you may have a couple of duplicate entries, otherwise LGTM.

File: contrib/completions/bash/buildah (hunk 1 above)
Comment (rhatdan) [reply contrib/completions/bash/buildah]: I believe we only need the buildah_buildah() ones, the other global options in buildah() should be removed.

Comment (q384566678) [reply contrib/completions/bash/buildah]: updated. It's strange why tests fail.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**